game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyServerProperties.java

```diff
@@ -27,7 +29,8 @@
   /** The port the lobby is listening on. */
   @Nonnull private final Integer port;
 
-  @Nullable private final String version;
+  /** URI the http lobby server. */
```

Comment:
`The http lobby server URI.` ?

Reply:
Makes more sense with all the words there :grin: 

 "URI *for* the http..."  
90fdf94d5